tispark-overview.md

```diff
@@ -32,308 +32,407 @@ Also, TiSpark supports distributed writes to TiKV. Compared with writes to TiDB
+## Getting TiSpark
 
-- It is recommended to provision at least 8 to 16 cores on per machine for Spark. Initially, you can assign all the CPU cores to Spark.
+TiSpark is a third-party jar package for Spark that provides the ability to read/write TiKV
 
-### Configuration of co-deployed TiKV and TiSpark
+### Get mysql-connector-j
 
-To co-deploy TiKV and TiSpark, add TiSpark required resources to the TiKV reserved resources, and allocate 25% of the memory for the system.
+We will not provide the `mysql-connector-java` dependency because of the limit of the GPL license.
 
-## Deploy the TiSpark cluster
+The following versions of TiSpark's jar will no longer include `mysql-connector-java`.
 
-Download TiSpark's jar package [here](https://github.com/pingcap/tispark/releases) and place it in the `$SPARKPATH/jars` folder.
+- TiSpark > 3.0.1
+- TiSpark > 2.5.1 for TiSpark 2.5.x
+- TiSpark > 2.4.3 for TiSpark 2.4.x
 
-> **Note:**
->
-> TiSpark v2.1.x and older versions have file names that look like `tispark-core-2.1.9-spark_2.4-jar-with-dependencies.jar`. Please check the [releases page on GitHub](https://github.com/pingcap/tispark/releases) for the exact file name for the version you want.
+Now, TiSpark needs `mysql-connector-java` for writing and auth. Please import `mysql-connector-java` manually when you need to write or auth.
 
-The following is a short example of how to install TiSpark v2.4.1:
+- you can import it by putting the jar into spark jars file
 
-{{< copyable "shell-regular" >}}
+- you can also import it when you submit spark job like
 
-```shell
-wget https://github.com/pingcap/tispark/releases/download/v2.4.1/tispark-assembly-2.4.1.jar
-mv tispark-assembly-2.4.1.jar $SPARKPATH/jars/
 ```
+spark-submit --jars tispark-assembly-3.0_2.12-3.1.0-SNAPSHOT.jar,mysql-connector-java-8.0.29.jar
+```
+
+### Choose TiSpark Version
+
+You can choose TiSpark version according to your TiDB and Spark version
+
+| TiSpark version | TiDB、TiKV、PD version | Spark version | Scala version |
+| ---------------  | -------------------- | ------------- | ------------- |
+| 2.4.x-scala_2.11 | 5.x, 4.x             | 2.3.x, 2.4.x   | 2.11          |
+| 2.4.x-scala_2.12 | 5.x, 4.x             | 2.4.x         | 2.12          |
+| 2.5.x            | 5.x, 4.x             | 3.0.x, 3.1.x   | 2.12          |
+| 3.0.x            | 5.x, 4.x             | 3.0.x, 3.1.x, 3.2.x|2.12|
+| 3.1.x            | 6.x, 5.x, 4.x             | 3.0.x, 3.1.x, 3.2.x, 3.3.x|2.12|
+
+- TiSpark 2.4.3, 2.5.2, 3.0.2,3.1.0 is the latest stable version, which is highly recommended.
 
-Copy the `spark-defaults.conf` from the `spark-defaults.conf.template` file:
+### Get TiSpark jar
 
-{{< copyable "shell-regular" >}}
+- get from [maven central](https://search.maven.org/) and search with GroupId [![Maven Search](https://img.shields.io/badge/com.pingcap-tikv/tispark-green.svg)](http://search.maven.org/#search%7Cga%7C1%7Cpingcap)
```

Comment:
Is this badge right? Why it's `tikv/tispark`?

Reply:
I will try to get a new one